CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixes
+
+- The SDK no longer closes the underlying native SDK during the games shutdown if native support has not been enabled. This enables the SDK to support and capture errors in case of building the game as a library. ([#1897](https://github.com/getsentry/sentry-unity/pull/1897))
```

Comment:
```suggestion
- The SDK no longer closes the underlying native SDK during the games shutdown if native support has not been enabled. This allows the SDK to support and capture errors in case of building the game as a library on a mobile (Android or iOS) game. ([#1897](https://github.com/getsentry/sentry-unity/pull/1897))
```